rpcs3/Emu/RSX/Overlays/overlay_trophy_notification.cpp

```diff
@@ -133,8 +136,8 @@ namespace rsx
 			text_view.auto_resize();
 
 			// Resize background to cover the text
-			u16 margin_sz = text_view.x - image.w - image.x;
-			frame.w       = text_view.x + text_view.w + margin_sz;
+			u16 margin_sz = 9;
+			frame.w       = 72 + text_view.w + margin_sz;
```

Comment:
why are these values suddenly hardcoded?
the logic behind it should not change.

Reply:
Because this logic generates a giant right margin. I could make new logic if padding worked as it should, but it doesn't.
The best i can do is leave the first line like this and replace the second line with:
frame.w = margin_sz * 3 + image.w + text_view.w;

Reply:
That doesn't make sense. It never created a giant right margin before, so why should it now?